### PR TITLE
[10.0][MIG] sale_stock

### PR DESCRIPTION
**Pull request by reinka**
This PR adds the migration scripts to v10 for the sale_stock module.

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/sale_stock/migrations/10.0.1.0/openupgrade_analysis_work.txt [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---Fields in module 'sale_stock'---
+sale_stock   / stock.move               / to_refund_so (boolean)        : NEW
```
Comment (pedrobaeza) [inline addons/sale_stock/migrations/10.0.1.0/openupgrade_analysis_work.txt:2]: Please marked as default this field, as if you come from v9, you expect that all the already made moves are refundable.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Add in the documentation (https://github.com/OCA/OpenUpgrade/blob/10.0/odoo/openupgrade/doc/source/modules90-100.rst) the "Done" word for this module.

File: addons/sale_stock/migrations/10.0.1.0/openupgrade_analysis_work.txt (hunk 1 above)
Comment (reinka) [reply addons/sale_stock/migrations/10.0.1.0/openupgrade_analysis_work.txt:2]: @pedrobaeza Update according to your request

**Review by pedrobaeza — APPROVED** [review]

**Comment by jholze** [comment]
👍

**Comment by StefanRijnhart** [comment]
@pedrobaeza please don't merge modules of which the dependencies have not yet been merged because it makes it impossible for us to get Travis green.

**Pull request merged**